src/pyhub/llm/mcp/multi_client.py

```
@@ -2,7 +2,8 @@
 
 import asyncio
 import logging
-from typing import Any, Dict, List, Union
+from typing import Any, Dict, List
+import uuid, Union
```

Ruff (0.11.9)

Multiple imports on one line

Split imports

(E401)